packages/db/src/methods/removeWorkspaceById.js

```diff
@@ -0,0 +1,15 @@
+const R = require('ramda')
+
+const { deleteOneFrom } = require('./helpers')
+
+/**
+ *
+ */
+
+function removeWorkspaceById (knex, id) {
+  return deleteOneFrom(knex, 'workspaces', { id })
```

Comment:
consider defining table names somewhere else as constant and use like `tableNames.users`